### PR TITLE
Upgrade @edx/frontend-component-footer, add enterpriseMarketingLink and socialLinks props

### PR DESCRIPTION
**Pull request by adamstankiewicz**
Jira: [ENT-1605](https://openedx.atlassian.net/browse/ENT-1605)

Upgrades the @edx/frontend-component-footer NPM package, adds the `enterpriseMarketingLink` and `socialLinks` prop, both recently introduced.

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -86,14 +127,17 @@ const App = () => (
...
-          twitterUrl={process.env.TWITTER_URL}
-          youTubeUrl={process.env.YOU_TUBE_URL}
-          linkedInUrl={process.env.LINKED_IN_URL}
-          googlePlusUrl={process.env.GOOGLE_PLUS_URL}
-          redditUrl={process.env.REDDIT_URL}
           appleAppStoreUrl={process.env.APPLE_APP_STORE_URL}
           googlePlayUrl={process.env.GOOGLE_PLAY_URL}
+          socialLinks={socialLinks}
+          enterpriseMarketingLink={{
+            url: process.env.ENTERPRISE_MARKETING_URL,
```
Comment (staubina) [inline src/index.jsx]: Do we need to add these to the local configuration and also our Stage and Prod configs?

Comment (adamstankiewicz) [reply src/index.jsx]: @christopappas and I have created an edx-internal PR for adding these configs on stage/prod for all the MFEs we're touching.

Good call on the local configuration. Missed that in this repo; adding it now.

Comment (christopappas) [reply src/index.jsx]: yeah, at this point, all the relevant stage and prod config variables and values have been merged to internal

Comment (staubina) [reply src/index.jsx]: 👍

**Pull request merged**